frontend/src/features/account/IENotSupportedPage.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Col, Container, Row } from 'react-bootstrap';
+
+export const IENotSupportedPage: React.FC = () => {
+  return (
+    <Container className="unauth" fluid={true}>
+      <h1>PIMS does not support Internet Explorer</h1>
```

Comment:
Technically it just doesn't support older versions of IE right? is this text correct?

Reply:
@devinleighsmith I thought it wasn't supporting IE at all